packages/wdio-cucumber-framework/tests/error-handling.test.js

```diff
@@ -0,0 +1,118 @@
+import path from 'path'
+import { CucumberAdapter } from '../src'
+
+const specs = [path.join(__dirname, '/fixtures/sample.feature')]
+
+const NOOP = () => {}
+
+process.send = NOOP
+
+// what are '/fixtures/async-step-definitions.js' and '/fixtures/async-step-definitions.js'
+describe.skip('ignores service hook errors', () => {
```

Comment:
let's make this test work before proceeding. Can anyone help, please?!

Reply:
after deeper investigation I think this test doesn't make sense. Cases mentioned there are covered by `executeHooksWithArgs` function. It is enough to verify that function is called and we have such test - `hookRunner.test.js`